tracker-radar-tests/TR-domain-matching/surrogates.txt

```diff
@@ -0,0 +1,6 @@
+# This file contains "surrogates". Surrogates are small scripts that our apps and extensions serve in place of trackers that cause site breakage when blocked.
+# Learn more: https://github.com/duckduckgo/tracker-surrogates
+surrogates.test/tracker application/javascript
+(function() {
```

Comment:
This doesn't match the js version, it's 

```
(function() {
    var tracker=true
})();
```

vs 

`(function() {var tracker=true})();`

Since the test definition has a specific string encoded in base64 (the version of the script without newlines), the tests fail.

I could add some fuzziness to the tests to ignore newlines when comparing strings, if we're supposed to ignore newline differences.

Reply:
🤦‍♂️ thanks for checking @RendijsSmukulis , you are absolutely right, they should match. I overdid it with the formatting. It's updated now.